pkg/mtu/detect_linux.go

```diff
@@ -65,6 +66,9 @@ func autoDetect() (int, error) {
 	}
 
 	if mtu := link.Attrs().MTU; mtu != 0 {
+		if mtu < maxCiliumMTU {
```

Comment:
isn't this the reverse of what the commit message is trying to say?

Reply:
typo thanks.